spec/features/admin/firm_approval_spec.rb

```diff
@@ -33,9 +35,16 @@ def given_i_am_at_the_admin_firms_index_page
   end
 
   def given_there_are_fully_registered_principal_users_with_advisers_and_offices
```

Comment:
@thenapking I'm getting a failure in this test when I run it locally. Not sure if it's a data issue or something. 
![image](https://user-images.githubusercontent.com/6080548/105483960-87ef5b00-5ca2-11eb-8f6f-5a117c28bfe0.png)